LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_ThermoCouple.TcPOU

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.12">
+  <POU Name="FB_ThermoCouple" Id="{cc00895a-0264-4382-820e-9cfa9062e545}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_ThermoCouple
+(* Handles scaling and default diagnostics for thermocouple terminals *)
+VAR_INPUT
+	// Ratio between raw value and actual temperature. Default is 10 for 10 steps per degree (or 0.1 degree resolution)
+	iScale: INT := 10;
+END_VAR
+VAR_OUTPUT
+	{attribute 'pytmc' := '
+		pv: TEMP
+		io: input
+	'}
+	fTemp: LREAL;
+	
+	{attribute 'pytmc' := '
+		pv: CONN
+		io: input
+		ONAM: Connected
```

Comment:
```suggestion
		field: ONAM Connected
```

Reply:
Thanks, would I do this again the same way for the ZNAM?

Reply:
Yes, the same.

We could perhaps change the convention - if all uppercase, assume a field. I kinda like that, actually (@n-wbrown?)

Reply:
I kind of like that idea too, but I also like how the current syntax is very explicit.